public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.js

```diff
@@ -118,18 +150,29 @@ class SpeakersBlockContent extends Component {
 												<a
 													className={ 'wordcamp-speaker-session-link' }
 													href={ session.link }
+													// is there an esc_url() equivalent in g? or do we not need it because we're returning an Element, which gets escaped automatically?
 												>
 													{ decodeEntities( session.title.rendered.trim() ) || __( '(Untitled)', 'wordcamporg' ) }
 												</a>
 												<br />
 												<span className={ 'wordcamp-speaker-session-info' }>
 													{ session.session_track.length &&
+														// will this fatal similar to https://meta.trac.wordpress.org/changeset/8025/ ?
+														// if so, it'd be good to check for other instances of that problem throughout the plugin
+
```

Comment:
Good question. I'll have to test, as I'm not sure if the REST API strips out the key if the value is empty.